pkg/controller/operators/olm/operator.go

```diff
@@ -1361,15 +1372,14 @@ func getCopiedCSVsCondition(isDisabled, csvIsRequeued bool) metav1.Condition {
 		return condition
 	}
 
+	condition.Reason = "CopiedCSVsDisabled"
```

Comment:
Does this need to live as a constant? Where are the reasons typically defined, o-f/api?

Reply:
That's probably a good idea, any objections to doing this in a followup PR if I create an issue to track it?

Reply:
Just saw this: that's fine with me, but I'd like to see us tackle this sooner than later.